params/extras/config_extra.go

```diff
@@ -0,0 +1,261 @@
+// (c) 2024 Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package extras
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/big"
+
+	"github.com/ava-labs/avalanchego/snow"
+	"github.com/ava-labs/libevm/common"
+	ethparams "github.com/ava-labs/libevm/params"
+)
+
+// XXX: Must fix this
+var GetExtra func(c *ethparams.ChainConfig) *ChainConfigExtra
+
+// UpgradeConfig includes the following configs that may be specified in upgradeBytes:
+// - Timestamps that enable avalanche network upgrades,
+// - Enabling or disabling precompiles as network upgrades.
+type UpgradeConfig struct {
+	// Config for enabling and disabling precompiles as network upgrades.
+	PrecompileUpgrades []PrecompileUpgrade `json:"precompileUpgrades,omitempty"`
+}
+
+// AvalancheContext provides Avalanche specific context directly into the EVM.
+type AvalancheContext struct {
+	SnowCtx *snow.Context
+}
+
+type ChainConfigExtra struct {
```

Comment:
(nit) Since this is now in the `extras` package and will always be accessed like that, the `Extra` suffix is unnecessary.

`extras.ChainConfig` vs `extras.ChainConfigExtra`

Reply:
Sure, will rename it.